packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -40,26 +40,21 @@ export class H5WasmApi extends ProviderApi {
   }
 
   public async getEntity(path: string): Promise<Entity> {
-    const file = await this.file;
-    const h5wEntity = file.get(path);
-
+    const h5wEntity = await this.getH5WasmEntity(path);
```

Comment:
We now need to check that the entity returned is defined and throw an entity-not-found error if it isn't, so I've moved this to a separate method.